### ISSUE TITLE
Language pack installation issue when another version with the same target locale was previously installed

### ISSUE THREAD

**Issue by AlexandraMoga** [mozilla/addons-frontend#7615]
Prerequisites: set the required prefs in about:config in order to enable installation from the -dev server

**STR**

1. Open the following language pack and install it - https://addons-dev.allizom.org/en-US/firefox/addon/korean-kr-languagepack/
2. Try to also install this language pack - https://addons-dev.allizom.org/en-US/firefox/addon/korean-kr-lpack/

**Actual result:**
The second lang pack appears to be installed and the button state changes to Remove. However, when looking in addons manager you will see an "Installation failed" message next to this language pack. Also, you won't be able to remove the language pack completely, unless you delete it from the `extensions` directory of your current profile.
The console is throwing the following error:
`Error: Source with name "toolkit-langpack-ko-browser" already registered.`

**Expected result:**
Show an error in AMO that will reflect what is reported by the console

**Notes:**
- I suppose this is a low priority issue, given that the main source of lang pack installations is our AMO language tools page, where there are no target_locale duplications
- reproduced on all AMO servers with FF65, Win10x64

![2019-02-20_16-53-29](https://user-images.githubusercontent.com/31961530/53100648-c3220600-3530-11e9-86ff-2cd41df5dfe0.gif)

**Comment by jvillalobos** [comment]
Language pack duplication shouldn't be a problem in prod now.
